lib/faraday/adapter/em_http.rb

```diff
@@ -280,7 +280,7 @@ def check_finished
   rescue LoadError
     warn 'Warning: no such file to load -- openssl. ' \
       'Make sure it is installed if you want HTTPS support'
-  else
+  elsif EventMachine::HttpRequest::VERSION < '1.1.6'
```

Comment:
Ah, `begin rescue else` does not support an `elsif`, it will need to be an `if` inside the `else`.

Reply:
Oops - I'll get that switched